### PR TITLE
翻譯

### PR DESCRIPTION
**Pull request by godchadigo**

### CODE REVIEW

File: MODBUS-LIB/Inc/Modbus.h [hunk 1]
```diff
@@ -74,6 +74,25 @@ uint8_t u8available;
...
+	* uint8_t u8lastRec;
+	* uint16_t *au16regs;
+	* uint16_t u16InCnt, u16OutCnt, u16errCnt; //保留Modbus流量的統計信息 
+	* uint16_t u16timeOut;
+	* uint32_t u32time, u32timeOut, u32overTime;
+	* uint16_t u16regsize;
+	* uint8_t dataRX;
+	* int8_t i8state;
+	*/
+
```
Comment (godchadigo) [inline MODBUS-LIB/Inc/Modbus.h:95]: 功能註解

Comment (alejoseb) [reply MODBUS-LIB/Inc/Modbus.h:95]: Hi, 
Thanks for your translation  and annotations in Chinese, however I can only maintain this library in English.

I think your annotation is still very important for Chinese speakers. I think I can add a second README file with your translation and you can add your comments and annotation there explaining the fields of the Modbus handler.  
I will be more than happy to add a second README file in Chinese to the master repo.
Meanwhile I will not merge this pull request.

File: README.md [hunk 2]
```diff
@@ -87,4 +87,80 @@ Windows: https://sourceforge.net/projects/modrssim2/
...
+
+Windows: https://sourceforge.net/projects/modrssim2/
+* * *
+# 目標
+* 為主功能代碼實現包裝器功能。 當前，telegrams是手動定義的。 
+* 改進功能文檔 
+* ~~改善數據接收隊列； 當前方法過於繁重，應將其替換為簡單的緩衝區，流或其他FreeRTOS原語。 已解決的隊列由環形緩衝區重新填充 [Ring Wiki](https://zh.wikipedia.org/wiki/%E7%92%B0%E5%BD%A2%E7%B7%A9%E8%A1%9D%E5%8D%80)（03/19/2021)~~
+* ~~使用Rs485收發器進行測試（已實現，但未經測試）已通過MAX485收發器進行了驗證（01/03/2021)~~
+* Modbus TCP功能實現
+* * *
```
Comment (godchadigo) [inline README.md:166]: New Translation File .

**Review by alejoseb — CHANGES_REQUESTED** [review]
Please add a new file for the translation, but do not add the translation at the end of the original Readme file. You can add a link t in the original Readme pointing to the Chinese translation file.  
Also, please remove the Chinese anotations from  Modbus.h, instead you can extend the translation file adding the annotation there.

**Comment by alejoseb** [comment]
I solved the issues and merged your readme file. Thanks!

**Pull request merged**